.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "openQA Single Instance",
+    "image": "registry.opensuse.org/devel/openqa/containers15.5/openqa-single-instance",
```

Comment:
With this we would need to update this definition file here and in the other places where you added a dev container definition on every new release of Leap. Either we write a checklist what are all the places to update on Leap release or we use a version agnostic path. Should we wait for https://progress.opensuse.org/issues/161399 and give that ticket more focus?

Reply:
Makes little sense to block here imho. This works. We can change it once the underlying issue is resolved.

Reply:
Yes, we can. My point is: How do we remember/ensure that we change it once the underlying issue is resolved

Reply:
I would think we eventually get to the ticket since we want to resolve [poo#48641](https://progress.opensuse.org/issues/48641) (the epic) - alternatively put [poo#161399](https://progress.opensuse.org/issues/161399) on the backlog to make it trigger our SLO and effectively make me check it.

Reply:
@kalikiana https://progress.opensuse.org/issues/161399 does not mention that the devcontainer in https://github.com/os-autoinst/os-autoinst-distri-opensuse/ or the other places where it's already used would need to be updated to Leap 15.5. So from experience whoever would work on https://progress.opensuse.org/issues/161399 would likely make sure that a Tumbleweed container works in Codespaces, *maybe* remember to also change our container build definition in OBS but *likely not* remember to update all devcontainer definitions in various test distribution repositories unless explicitly stated in a checklist.